src/Protocol/ActivityPub/Processor.php

```diff
@@ -359,9 +359,7 @@ private static function processContent($activity, $item)
 					return false;
 				}
 
-				$potential_implicit_mentions = self::getImplicitMentionList($parent);
-				$content = self::removeImplicitMentionsFromBody($content, $potential_implicit_mentions);
-				$activity['tags'] = self::convertImplicitMentionsInTags($activity['tags'], $potential_implicit_mentions);
```

Comment:
By what has this been replaced with?

Reply:
We don't need `convertImplicitMentionsInTags` anymore because the new tag systematic automatically detects the type of the mention. With `removeImplicitMentionsFromBody` the mention is removed from the body, so that mention will be stored as implicit mention.

Reply:
Ok.